Sources/Networking/AlamofireNetworkClient/AlamofireNetworkClient.swift

```diff
@@ -174,12 +202,9 @@ public extension AlamofireNetworkClient.Request {
     }
   }
   
-  func decode<D: Decodable>(
-    _ decodable: D.Type,
-    decoder: JSONDecoder) -> Promise<D>
-  {
+  func decode<D: Decodable>(_ decodable: D.Type, decoder: JSONDecoder? = nil) -> Promise<D> {
```

Comment:
Should we follow the same style throughout the file?

Reply:
We are not currently. There are some other methods that don't follow the same indentation rule. But I can make it if you prefer that way 🤷‍♂️